IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedFacilitiesElements/PropertySets/Pset_Asset/DocPropertySet.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_Asset" UniqueId="23KsS0qS4HuO00025QrE$V" ApplicableType="IfcAsset" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_Asset" UniqueId="23KsS0qS4HuO00025QrE$V" ApplicableType="IfcAsset,IfcSystem,IfcElement" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
applicability should be reviewed. the whole concept of IfcAsset is to put elements into the asset and give it properties. why put the properties on the element and system also?

Reply:
I understand the principle of IfcAsset, as a grouping of things together as a singular asset, and it has its uses when this is the case, but it also means when you have a single entity that has to get asset information you have to add it to a group of one for the asset then add the extra Pset info to store all the information. I fine this counter intuative when all this information can be within a property set and linked directly to the entity, shortening the chain.

that is my opinion, but if this is a bigger issue we can revert to the original form.

Reply:
It conflicts with the Pset name which is counterintuitive from a different perspective. Also, we would have to address the issue with documentation:
"An asset is a group that can contain one or more elements."

Reply:
Yes, I understand that IfcAsset 

> is a group that can contain one or more elements

but as my point before, using the 'one' part is bloat when property sets is the existing mechanism for this.

we have an asset manager on our project and they tag this information all the way down to individual ladders and bollards , hence my concern with bloat.

Reply:
I do not think it has a big influence on bloating the files (not sure if you are talking about file size). Usually, geometry and individual properties take the biggest chunk, 40-50% and10-40% respectively. I see a big benefit in semantical clarity though.

Reply:
I was refering to file size. i understand geometry forms a large portion of the size. but i always try to minimise the file size where ever i can, i agree with the semantical clarity, i think i will take the same view as Pset_Risk, we will stick with the current way connecting to IfcAsset and leave it as a discussion further down the line